scripts/getting-started.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/sh
+
+prompt() {
+    while true; do
+        echo "$1 [y/N]"
+        read yn
+        case $yn in
+            [Yy]* ) return 0;;  # Yes, return 0 (true)
+            [Nn]* ) return 1;;  # No, return 1 (false)
+            "" ) return 1;;     # Default to no if user just presses Enter
+            * ) echo "Please answer yes or no.";;
+        esac
+    done
+}
+
+cat <<EOF
+
+ Welcome to the
+
+     , __       _   _                          ____  ____  _  __
+    /|/  \     | | | |           |            / ___||  _ \| |/ /
+     |___/ __  | | | |   __,   __|   __ _|_   \___ \| | | | ' / 
+     |    /  \_|/  |/_) /  |  /  |  /  \_|     ___) | |_| | . \ 
+     |    \__/ |__/| \_/\_/|_/\_/|_/\__/ |_/  |____/|____/|_|\_\ 
+                                                                    quickstart!
+
+🚀 We will be setting up an example template and its environment for you to experiment with.
+EOF
+
+# Determine OS
+os_name=$(uname -s)
+if [ "$os_name" = "Darwin" ]; then
+    echo "🍎 Detected macOS. Installing dependencies via Homebrew."
+
+    # Check if brew is installed
+    if command -v brew >/dev/null 2>&1; then
+        echo "\n✅︎🍺 Homebrew already installed."
+    else
+        if prompt "\n🍺 Homebrew is not installed. Install it?"; then
+            echo "🍺 Installing Homebrew."
+            /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

Comment:
I think wget is default installed on Debian, but not curl. Maybe do a sanity check at the top that all the requirements are there with a nice error message otherwise?

Reply:
This is installing homebrew, though? It's on MacOS

Reply:
It is also further down used. I just tried this in a docker and got the error that `wget` was not found.